sched/clock/clock_gettime.c

```diff
@@ -58,15 +58,15 @@ static clock_t clock_process_runtime(FAR struct tcb_s *tcb)
 
   group = tcb->group;
 
-  flags = spin_lock_irqsave(NULL);
+  flags = spin_lock_irqsave(&group->lock);
```

Comment:
replace all spin_lock_irqsave to spin_lock_irqsave_wo_note in sched dir

Reply:
Will be completed in the next patch

Reply:
I suggest you change this patch, otherwise there may be deadlock problems

Reply:
done